helm/opendj/values.yaml

```diff
@@ -120,11 +120,12 @@ restore:
   # The most recent backup will be restored. 
   enabled: false
  
+# DS proxy - experimental.
 proxy:
   enabled: false
 
 haproxy:
-  enabled: true
+  enabled: false
   replicas: 1
```

Comment:
Would we need a replica for proxy ever? if not could we not get rid of haproxy.replicas